processing/data_collection/gazette/spiders/mt_cuiaba.py

```diff
@@ -0,0 +1,54 @@
+import json
+import time
+from dateparser import parse
+from datetime import datetime
+from scrapy import Request
+from scrapy.downloadermiddlewares.retry import RetryMiddleware
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+# based on the website
+FIRST_YEAR = 1967
+BASE_URL = "https://diariooficial.cuiaba.mt.gov.br/api/api/editions"
+
+
+class MtCuiabaSpider(BaseGazetteSpider):
+    TERRITORY_ID = 5103403
+    name = "mt_cuiaba"
+    allowed_domains = ["diariooficial.cuiaba.mt.gov.br"]
+    start_urls = ["https://diariooficial.cuiaba.mt.gov.br/"]
```

Comment:
Is this still valid? I'm trying to access but no success. Furthermore, when I run the spider I can see:

```
2020-09-15 18:22:03 [scrapy.downloadermiddlewares.retry] DEBUG: Retrying <GET https://diariooficial.cuiaba.mt.gov.br/api/api/editions/published/1967/4> (failed 2 times): DNS lookup failed: no results for hostname lookup: diariooficial.cuiaba.mt.gov.br.
2020-09-15 18:22:05 [scrapy.downloadermiddlewares.retry] DEBUG: Retrying <GET https://diariooficial.cuiaba.mt.gov.br/api/api/editions/published/1967/6> (failed 2 times): DNS lookup failed: no results for hostname lookup: diariooficial.cuiaba.mt.gov.br.
2020-09-15 18:22:07 [scrapy.downloadermiddlewares.retry] DEBUG: Retrying <GET https://diariooficial.cuiaba.mt.gov.br/api/api/editions/published/1967/7> (failed 2 times): DNS lookup failed: no results for hostname lookup: diariooficial.cuiaba.mt.gov.br.
2020-09-15 18:22:07 [scrapy.downloadermiddlewares.retry] DEBUG: Retrying <GET https://diariooficial.cuiaba.mt.gov.br/api/api/editions/published/1967/8> (failed 2 times): DNS lookup failed: no results for hostname lookup: diariooficial.cuiaba.mt.gov.br.

```

I think the new URL is `http://gazetamunicipal.cuiaba.mt.gov.br` Can you double check that?